Code/Tools/SceneAPI/SceneBuilder/Importers/AssImpMaterialImporter.cpp

```diff
@@ -175,17 +175,36 @@ namespace AZ
                 return combinedMaterialImportResults.GetResult();
             }
 
-            AZStd::string AssImpMaterialImporter::ResolveTexturePath(const AZStd::string& sceneFilePath, const AZStd::string& textureFilePath) const
+            AZStd::string AssImpMaterialImporter::ResolveTexturePath(const AZStd::string& materialName, const AZStd::string& sceneFilePath, const AZStd::string& textureFilePath) const
             {
                 if (textureFilePath.empty())
                 {
+                    AZ_TracePrintf(AZ::SceneAPI::Utilities::LogWindow, "Material %.*s has no associated texture.", AZ_STRING_ARG(materialName));
                     return textureFilePath;
                 }
 
-
-                AZStd::string texturePathRelativeToScene;
                 AZStd::string cleanedUpSceneFilePath(sceneFilePath);
                 AZ::StringFunc::Path::StripFullName(cleanedUpSceneFilePath);
+                
+                AZ::IO::PathView textureFilePathView(textureFilePath.c_str());
+
+                if (textureFilePathView.IsAbsolute())
+                {
+                    // Don't try to resolve the absolute path of the texture relative to the scene file,
+                    // because it may resolve differently on different machines if the path happens to be
+                    // correct on one person's machine but not another.
+                    // Example: Texture is at C:\path\to\MyProject\assets\MyTexture.tga.
+                    // On one person's machine, if their scene file is in that folder, example C:\path\to\MyProject\assets\MyFBX.fbx
+                    // then comparing the path of the scene file and texture might properly resolve here.
+                    // However, if a second person on the team's machine has the project at D:\path2\to2\MyProject,
+                    // the same logic won't resolve.
+                    AZ_TracePrintf(AZ::SceneAPI::Utilities::LogWindow,
+                        "Material %.*s has a texture with absolute path '%.*s'. This path will not be resolved.",
```

Comment:
"This path will not be resolved." Maybe explain is this will cause issues or not?

Reply:
Sure, I've updated the log to explain what happens.